packages/components/src/Button/PaginationLink/PaginationLink.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react"
+import { GenericProps, GenericButton } from "../GenericButton"
+
+export interface PaginationLinkProps extends GenericProps {
+  pageNumber: number
+  isActive?: boolean
+  /* A description for screen readers to understand because label used here is a number */
+  "aria-label": string
+}
+
+/**
+ * {@link https://cultureamp.atlassian.net/wiki/spaces/DesignSystem/pages/3082092975/Pagination Guidance} |
+ * {@link https://cultureamp.design/?path=/story/components-buttons-paginationlink--docs Storybook}
+ */
+export const PaginationLink = (props: PaginationLinkProps): JSX.Element => (
+  <GenericButton
+    {...props}
```

Comment:
You actually need to destructure `pageNumber` because it gets passed into `label`, which means it being part of the spread props will mean `GenericButton` has no idea what `pageNumber` is and try to insert it into the dom.

```suggestion
export const PaginationLink = ({ pageNumber, ...restProps }: PaginationLinkProps): JSX.Element => (
  <GenericButton
    {...restProps}
```

Reply:
How did this work before...?

Reply:
You didn't fix this?

Reply:
Oh, must've gotten lost